docs/settings_dict_guide.md

```diff
@@ -14,17 +14,14 @@ tags:
 This document enumerates all the settings and configuration options available when
 developing your data linkage model.
 
-You can find an interative settings editor [here](https://moj-analytical-services.github.io/splink/settingseditor/editor.html).
+You can find an interative settings editor [here](/docs/settingseditor/editor.md).
 
-## Settings keys in the base setting dictionary
+<hr>
 
-
-### link_type
+## link_type
 
 The type of data linking task.  Required.
```

Comment:
Not in this PR, but I think the _**Required.**_ statement should probably be in italics alongside the `## link_type` title

Reply:
I have tried adding these to all of the parameters - I don't think I like it as it really clutters up the table of contents making the page harder to navigate